src/code/stores/app-settings-store.coffee

```diff
@@ -3,6 +3,7 @@ AppSettingsActions = Reflux.createActions(
     "showSettingsDialog"
     "close"
     "capNodeValues"
+    "importSettings"
```

Comment:
If we are following the actions pattern (don't feel strongly about this) then the method `importSettings` maybe should be renamed `onImportSettings`.